libraries/AP_HAL_PX4/bxcan.h

```diff
@@ -150,21 +49,6 @@ namespace bxcan {
 #define GPIO_CAN2_TX    GPIO_CAN2_TX_2
 #endif
```

Comment:
I understand they are pin specific, that means they are board specific. They are defined in the board header for each board, for example FMUv2: https://github.com/ArduPilot/PX4Firmware/blob/master/nuttx-configs/px4fmu-v2/include/board.h#L236-L239